lib/src/values/config_number.cc

```diff
@@ -31,4 +37,32 @@ namespace hocon {
     bool config_number::operator!=(const config_number &other) const {
         return !(*this == other);
     }
+
+    int config_number::int_value_range_checked(std::string path) {
```

Comment:
What's this used for, why does it take a path argument, and path should be passed by const&.